### PR TITLE
Fix mousewheel eventname on IE11<=

### PR DESCRIPTION
**Pull request by lorenzomigliorero**
## Description
The only way to test support for the wheel event in IE9+ is check the feature `Events.wheel` on the `document.implementation.hasFeature` object, instead of `window`. ([see here](https://github.com/basilfx/normalize-wheel/blob/a1b69f0096ee4fca1436241e493f7d7919242cc7/src/isEventSupported.js#L56))

Without this test, the eventname on IE11< is `mousewheel`, instead of `wheel`:
`const eventName = 'onwheel' in window ? 'wheel' : 'mousewheel' // mousewheel ❌`
`const eventName = 'onwheel' in window || document.implementation.hasFeature('Events.wheel', '3.0') ? 'wheel' : 'mousewheel' // wheel 👍`

Using `wheel` event, wheel data on IE11< is more similar than other browser, with benefit for the user experience.

The package `normalize wheel` normalize data across browsers and OS in a more accurate way.

## Types of changes
- [X] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Comment by idiotWu** [comment]
Alright, `normalize-wheel` package again 🤷‍♂️ 

Anyway I don't think the [delta normalizing function](https://github.com/basilfx/normalize-wheel/blob/a1b69f0096ee4fca1436241e493f7d7919242cc7/src/normalizeWheel.js#L141) in that package is correct. In my tests `evt.wheelDeltaY = -3 * evt.deltaY`, but in that package it's `evt.wheelDeltaY = -12 * evt.deltaY`.

image

File: src/events/wheel.ts [hunk 1]
```diff
@@ -1,57 +1,21 @@
...
-
-const DELTA_MODE = [1.0, 28.0, 500.0];
-
-const getDeltaMode = (mode) => DELTA_MODE[mode] || DELTA_MODE[0];
-
 export function wheelHandler(scrollbar: I.Scrollbar) {
   const addEvent = eventScope(scrollbar);
-
   const target = scrollbar.options.wheelEventTarget || scrollbar.containerEl;
+  const eventName = 'onwheel' in window || document.implementation.hasFeature('Events.wheel', '3.0') ? 'wheel' : 'mousewheel';
```
Comment (idiotWu) [inline src/events/wheel.ts]: Looks legit!

File: scripts/webpack.base.js [hunk 2]
```diff
@@ -41,5 +41,5 @@ module.exports = {
   ],
   stats: {
     modules: false,
-  },
+  }
```
Comment (idiotWu) [inline scripts/webpack.base.js]: Why these changes 😕

**Comment by lorenzomigliorero** [comment]
Pull request is about detection of mousewheel event on IE11, use normalize-wheel is secondary.

Using 'mousewheel' instead of 'wheel' in IE11 is wrong, scrolling delta normalized by your function is very low.
With 'wheel', wheeldelta is similar to other browser.
Please, consider to use this ternary for detect of events name:

const eventName = 'onwheel' in window || document.implementation.hasFeature('Events.wheel', '3.0') ? 'wheel' : 'mousewheel';

**Comment by idiotWu** [comment]
Yep I got your point. I just have no idea about the normalizing function in [normalize-wheel](https://github.com/basilfx/normalize-wheel/blob/master/src/normalizeWheel.js) package.

I must admit that the current normalizing function in this lib is a bit too simple. Maybe it's a good idea to use `normalize-wheel` package because the comments inside it are somehow reasonable 😆.

I just noticed that there's a [`getEventType` method](https://github.com/basilfx/normalize-wheel/blob/a1b69f0096ee4fca1436241e493f7d7919242cc7/src/normalizeWheel.js#L173-L179) in the package. Can we use it to get correct event name like:

```js
const eventName = normalizeWheel. getEventType();
```

**Comment by lorenzomigliorero** [comment]
Yes, i try to use `normalizeWheel. getEventType()`, but i obtain on Firefox 'DOMMouseScroll', which have wrong mousewheel data.

**Comment by idiotWu** [comment]
Hmmm...

Then what about fixing the event name only? We can discuss the matter with `normalize-wheel` later in another PR or issue if necessary.

**Comment by lorenzomigliorero** [comment]
Ok for fixing only the event name for now.

**Comment by idiotWu** [comment]
But I still can see a lot of deletion in files...

I think you just need to commit this line: https://github.com/idiotWu/smooth-scrollbar/pull/124/files#diff-481b84199b44eee82e410719c6df9216R11

File: scripts/webpack.base.js [hunk 3]
```diff
@@ -31,8 +31,8 @@ module.exports = {
   plugins: [
     new webpack.DefinePlugin({
       __SCROLLBAR_VERSION__: JSON.stringify(
-        process.env.SCROLLBAR_VERSION || require('../package.json').version,
-      ),
+        process.env.SCROLLBAR_VERSION || require('../package.json').version
+      )
```
Comment (idiotWu) [inline scripts/webpack.base.js]: and please exclude changes in this file too 😁

Comment (lorenzomigliorero) [reply scripts/webpack.base.js]: Ok for the second comma, but the comma after `version` cause this error:
image

Comment (idiotWu) [reply scripts/webpack.base.js]: So you're running on node <= 6? That would be fine with higher version of node.

**Comment by lorenzomigliorero** [comment]
It's ok? 😀
